src/common/utils.ts

```diff
@@ -0,0 +1,97 @@
+import turfIntersect from '@turf/intersect';
+import turfBBoxPolygon from '@turf/bbox-polygon';
+import turfLineStringToPolygon from '@turf/line-to-polygon';
+import { Geometry, Feature, Polygon, MultiPolygon, LineString, feature as turfFeature } from '@turf/helpers';
+import { BBox2d } from '@turf/helpers/dist/js/lib/geojson';
+import { Tile } from '../tile';
+import { COORDINATE_SYSTEM, TILE_AXIS_SIZE } from './constants';
+import { Coordinate } from './interfaces';
+
+let cacheZoomLevel: number;
+
+export function getTileResolution(zoomLevel: number): number {
+  if (cacheZoomLevel !== zoomLevel) {
+    cacheZoomLevel = COORDINATE_SYSTEM.maxLon / (1 << zoomLevel);
+  }
+  return cacheZoomLevel;
+}
+
+export function getPixelResolution(zoomLevel: number): number {
```

Comment:
try use memoization to improve preformance, for example: https://www.npmjs.com/package/memoizee

Reply:
Should we invest on doing so? All functions in `utils.ts` will be replaced by `mc-utils` package.

Reply:
ok i'll add it to the refactore issue